cypress/integration/find.spec.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable max-lines-per-function */
```

Comment:
I'm not sure this rule even makes sense in the context of a test file. Is the intent to limit how many tests there are in a file?

Reply:
Yeah, I don't use `describe` blocks much in my regular tests, but in Cypress it's kinda necessary. We can add an `.eslintrc` to the `cypress/` directory and disable it in there. Should probably disable the jest promise one too:

```javascript
{
  "rules": {
    "max-lines-per-function": "off",
    "jest/valid-expect-in-promise": "off"
  }
}
```

Reply:
That worked, thank you